arch/c4_node_containers.puml

```diff
@@ -0,0 +1,43 @@
+@startuml c4_node_container
+
+!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Container.puml
+
+!define Rel_Gossip(e_from, e_to, e_rel) Rel_(e_from, e_to, "gossips to", "UDP port 10000", e_rel)
+!define Rel_Sync(e_from, e_to, e_rel) Rel_(e_from, e_to, "syncs from", "UDP port 10000", e_rel)
+!define Rel_Client(e_from, e_to, e_rel) Rel_(e_from, e_to, "submits atoms to and recieves events from", "WebSocket port 443", e_rel)
+
+LAYOUT_WITH_LEGEND
+
+System_Ext(ws_app1, "WebSocket Client", "Custom Application that uses the WebSocket API directly or indirectly (lib).")
+
+System_Ext(inter_shard_node1, "Inter-Shard Node", "Other Core Node in the a **different** Shard space")
```

Comment:
Shouldn't it be called "Outer-Shard Node" then?

Reply:
Added shard boundaries and removed the intra/inter shard wording as I realised it causes confusion and is not consistent with the Context diagram.